lib/internal/bootstrap/loaders.js

```diff
@@ -99,7 +101,7 @@
     this.loading = false;
   }
 
-  NativeModule._source = getBinding('natives');
+  NativeModule._source = process.binding('natives');
```

Comment:
What't the difference between `getBinding` and `process.binding`?

Reply:
`process.binding` is legacy, and as mentioned in the comment at the beginning of `loaders.js`, they have made changes for embedders.
I'm not exactly sure how we can use linkedbinding, but as little as I know, `process.binding` and `getBinding` should work the same way.

Reply:
Honestly, I don't any reasons to make this change. Let's remove it )